PLCLocation.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collection.Generic;
+
+
+namespace mujinplccs{
+
+    public class Order{
+        public Order(){}
+        // some order finish status
+        public PLCLogic.PLCOrderCycleFinishCode orderCycleFinishCode;
+        public PLCLogic.PLCPreparationCycleFinishCode preparationCycleFinishCode;
+
+        // customer implemented interface
+        abstract public void onFinishCodeChangedCallback();
+        abstract public void onNumberPutInDestChangedCallback();
+        // order info
+        public string partType{get; set;}
+        public int orderNumber{get; set;}
+        public string uniqueId{get; set;}
+        public string containerId{get; set;}
+        public string containerType{get; set;}
+        public string robotId{get; set;}
+        public string placeContainerId{get; set;}
+        public string placeContainerType{get; set;}
+        public PLCLocation placeLocation{get; set;}
+        public PLCLocation pickLocation{get; set;}
+        public int packInputPartIndex{get; set;}
+        public string packFormationComputationName{get; set}
+    }
+    public class PLCLocation{ // class to represent location model
+        public PLCLocation(string locationName){
+            this.locationName = locationName;
+        }
+        abstract public void SetStatus();  // custom set this status to tell mujin controller if location is ready to pick
```

Comment:
Does integrator need to explicitly check `locationXReleased` before setting status, or this function would handle this automatically?

Reply:
Minor typo btw

Reply:
where's the argument? do you need to define a new type for it?